lib/datasource.js

```diff
@@ -826,7 +826,9 @@ DataSource.prototype.define = function defineClass(className, properties, settin
       settings.strict = true;
     }
     if (settings.strict === false) {
-      settings.strict = 'throw';
+      console.warn('WARNING: relational database doesn\'t support {strict: false} mode.' +
```

Comment:
Use double quotes to avoid escaping: "WARNING: relational database doesn't support {strict: false} mode."